src/NuGet.Core/NuGet.Packaging/Signing/Utility/SigningUtility.cs

```diff
@@ -172,16 +172,6 @@ public static CmsSigner CreateCmsSigner(SignPackageRequest request, ILogger logg
             return signer;
         }
 
-        public static async Task RemoveSignatureAsync(string packageFilePath, string outputFilePath, CancellationToken token)
```

Comment:
Why are you removing `RemoveSignatureAsync` ?

Reply:
It's a thin wrapper around `SignedPackageArchive.RemoveSignatureAsync(...)` and not used anywhere.